app/views/phases/_edit_plan_answers.html.erb

```diff
@@ -38,7 +38,7 @@
               <%= section.title %> 
               <% if plan.present? %>
                   <span class="section-progress-<%= sectionid %>">
-                    <%= render :partial => "/sections/progress", locals: { section: section, plan: plan } %>
+                    <%= render :partial => "/org_admin/sections/progress", locals: { section: section, plan: plan } %>
```

Comment:
this partial does not make sense to be located within org_admin folder since it is meant to address the progress of a plan section, i.e. use the default location for normal users and has nothing to do with org_admin namespacing.